src/Nethermind/Nethermind.Blockchain/Receipts/PersistentReceiptStorage.cs

```diff
@@ -51,6 +53,7 @@ public PersistentReceiptStorage(IColumnsDb<ReceiptsColumns> receiptsDb, ISpecPro
             _receiptsRecovery = receiptsRecovery ?? throw new ArgumentNullException(nameof(receiptsRecovery));
             _blocksDb = _database.GetColumnDb(ReceiptsColumns.Blocks);
             _transactionDb = _database.GetColumnDb(ReceiptsColumns.Transactions);
+            _logger = logManager.GetClassLogger(GetType());
```

Comment:
Is this logger ever used and needed?

Reply:
Actually no. It was used before for debugging, but the logs was removed.